### PR TITLE
Remove presets 

### PR DESCRIPTION
**Pull request by Amxx**
Replaces #3470

#### PR Checklist

- [ ] Tests
- [x] Documentation
- [x] Changelog entry

### CODE REVIEW

**Comment by BlackdoggInsane5150** [comment]
I need help can somebody  help me please 🙏

**Comment by frangio** [comment]
We need to remove presets from the documentation guides like `docs/modules/ROOT/pages/erc20.adoc`.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## Unreleased (Breaking)
 
  * `TimelockController`: Changed the role architecture to use `DEFAULT_ADMIN_ROLE` as the admin for all roles, instead of the bespoke `TIMELOCK_ADMIN_ROLE` that was used previously. This aligns with the general recommendation for `AccessControl` and makes the addition of new roles easier. Accordingly, the `admin` parameter and timelock will now be granted `DEFAULT_ADMIN_ROLE` instead of `TIMELOCK_ADMIN_ROLE`. ([#3799](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3799))
+* Remove presets in favor of the OZ Wizard.
```
Comment (frangio) [inline CHANGELOG.md:6]:
```suggestion
* Removed presets in favor of [OpenZeppelin Contracts Wizard](https://wizard.openzeppelin.com/).
```

**Review by frangio — COMMENTED** [review]
There seems to be a remaining preset for ERC721.

**Comment by JulissaDantes** [comment]
> There seems to be a remaining preset for ERC721.

Ready!

**Review by frangio — APPROVED** [review]

**Comment by BlackdoggInsane5150** [comment]
my apologies something went wrong im still learning the site

**Pull request merged**